lib/OpenQA/Task/Git/Clone.pm

```diff
@@ -50,9 +50,25 @@ sub _git_clone_all ($job, $clones) {
         my $url = $clones->{$path};
         die "Don't even think about putting '..' into '$path'." if $path =~ /\.\./;
         my $error;
-        try { _git_clone($app, $job, $ctx, $path, $url); next; }
+        try {
+            _git_clone($app, $job, $ctx, $path, $url);
+            OpenQA::Git::MaintenanceOutage::remove_state_file($app);
+            next;
+        }
         catch ($e) { $error = $e }
 
+        if ($error =~ /Internal API unreachable/) {
+            my $outage_handling = OpenQA::Git::MaintenanceOutage::decide_outcome($app, $ctx, $error);
+            if ($outage_handling->{skip}) {
+                $ctx->info('Git clone: GitLab outage likely, skipping clone job.');
```

Comment:
Please keep it generic. There should be nothing about gitlab here

Reply:
Yes i agree, though i'm not sure on how to word it, `Source Control Management System Outage` seemed rather long/weird to me, any better idea  ? :sweat_smile:

Reply:
Well, we are within "Git::MaintenanceOutage". How about "git server outage"?

Reply:
Renamed to `...Git server outage...` for both info texts